apps/nextjs/src/pages/eventupdate/EventUpdatePage.module.css

```diff
@@ -0,0 +1,3 @@
+.form_input {
```

Comment:
A minor nitpick on this, I would put this in the `apps/nextjs/src/styles` folder, do the import like `import styles from "../../styles/eventupdate.module.css";` instead for consistent logical group. Thoughts?